plugins/ssl/letsencrypt_weekly

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# monitor the number of CSRs by week for /etc/letsencrypt/csr/
```

Comment:
Please use a `perldoc` header.

Regarding the name of the plugin: maybe clarify the purpose of the plugin (e.g. *letsencrypt_certificate_request_count*)?

(maybe you want to make the *7 days* configurable later, thus I would not add the period to the plugin name)

Reply:
changed to perldoc

did not change the plugin name in this case (the weekly limit is named like this (weekly / per week) in the letsencrypt documentation)